save-cosv-frontend/src/main/kotlin/com/saveourtool/save/cosv/frontend/App.kt

```diff
@@ -0,0 +1,93 @@
+/**
+ * Main entrypoint for SAVE frontend
+ */
+
+package com.saveourtool.save.cosv.frontend
+
+import com.saveourtool.save.cosv.frontend.routing.basicRouting
```

Comment:
Do not forget to update COSV views before merging (there were changes from @DrAlexD on the frontend side of COSV). And looks like you have forgotten them

Reply:
Updated